### PR TITLE
Revert 233

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Revert the securityContext change since it caused permissions issues

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by danielezer** [comment]
@ankushchadha @jainishshah17 @rimusz please review. There's currently an issue after my latest change and the installations don't work because of FS permissions issues

**Comment by rimusz** [comment]
ok, let's revert this one back and then look how to make it work

File: stable/distribution/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # JFrog Distribution Chart Changelog
 All changes to this project chart be documented in this file.
 
-## [3.2.1] - Mar 13, 2019
-* Move securityContext to container level
+## [3.2.2] - Mar 15, 2019
```
Comment (jainishshah17) [inline stable/distribution/CHANGELOG.md]: @danielezer  We need 3.2.1's change log as well

File: stable/kubexray/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # JFrog KubeXray Chart Changelog
 All changes to this chart will be documented in this file.
 
-## [0.3.3] - Mar 13, 2019
-* Move securityContext to container level
+## [0.3.4] - Mar 15, 2019
+* Revert securityContext change that was causing issues
```
Comment (jainishshah17) [inline stable/kubexray/CHANGELOG.md]: @danielezer Same here add changelog for 0.3.3

File: stable/mission-control/CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
-## [0.9.2] - Mar 14, 2019
-* Move securityContext to container level
+## [0.9.3] - Mar 15, 2019
+* Revert securityContext change that was causing issues
```
Comment (jainishshah17) [inline stable/mission-control/CHANGELOG.md]: @danielezer same here

File: stable/distribution/CHANGELOG.md (hunk 1 above)
Comment (danielezer) [reply stable/distribution/CHANGELOG.md]: @jainishshah17 Done, thanks.

File: stable/artifactory-ha/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.11.10] - Mar 15, 2019
+* Revert securityContext changes since they were causing issues
```
Comment (jainishshah17) [inline stable/artifactory-ha/CHANGELOG.md]: @danielezer This one as well

File: stable/artifactory/CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [7.12.10] - Mar 15, 2019
+* Revert securityContext changes since they were causing issues
```
Comment (jainishshah17) [inline stable/artifactory/CHANGELOG.md]: @danielezer Same here

File: stable/kubexray/CHANGELOG.md (hunk 2 above)
Comment (danielezer) [reply stable/kubexray/CHANGELOG.md]: yup, done

File: stable/mission-control/CHANGELOG.md (hunk 3 above)
Comment (danielezer) [reply stable/mission-control/CHANGELOG.md]: Done

File: stable/artifactory/CHANGELOG.md (hunk 5 above)
Comment (danielezer) [reply stable/artifactory/CHANGELOG.md]: yup, it's there. version 7.12.6

File: stable/artifactory-ha/CHANGELOG.md (hunk 4 above)
Comment (danielezer) [reply stable/artifactory-ha/CHANGELOG.md]: same, version 0.11.6

**Pull request merged**